app/views/schools/participants/show.html.erb

```diff
@@ -43,33 +43,6 @@
           </dd>
         <% end %>
       </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          Start term
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= @profile.start_term.humanize %>
-        </dd>
-        <% unless @induction_record.training_status_withdrawn? %>
-          <dd class="govuk-summary-list__actions">
-            <% if @profile.policy_class.new(current_user, @profile).edit_start_term? %>
-              <%= govuk_link_to schools_participant_edit_start_term_path(participant_id: @profile) do %>
-                Change <span class="govuk-visually-hidden">start term</span>
-              <% end %>
-            <% end %>
-          </dd>
-        <% end %>
-      </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          Induction start
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= @first_induction_record&.start_date.to_date.to_s(:govuk) %>
-        </dd>
-        <dd class="govuk-summary-list__actions">
-        </dd>
-      </div>
```

Comment:
Should this be removed?

Reply:
Mili confirmed it needs to be removed

Reply:
Really? The induction start date bit too?

Reply:
it was a misunderstanding. The induction start should not be removed. I'll revert the change.
Thanks @tonyheadford 👍